src/utils/ExecsList.js

```diff
@@ -0,0 +1,160 @@
+import praneer from '../assets/execs/praneer.svg'
+import grace from '../assets/execs/grace.svg'
+import noble from '../assets/execs/noble.svg'
+import vivian from '../assets/execs/vivian.svg'
+import naomi from '../assets/execs/naomi.svg'
+import isshan from '../assets/execs/isshan.svg'
+import david from '../assets/execs/david.svg'
+import revika from '../assets/execs/revika.svg'
+import nargis from '../assets/execs/nargis.svg'
+import harry from '../assets/execs/harry.svg'
+import shubham from '../assets/execs/shubham.svg'
+import emma from '../assets/execs/emma.svg'
+import ali from '../assets/execs/ali.svg'
+import fiona from '../assets/execs/fiona.svg'
+import steven from '../assets/execs/steven.svg'
+import jocelyn from '../assets/execs/jocelyn.svg'
+import michelle from '../assets/execs/michelle.svg'
+import lauren from '../assets/execs/lauren.svg'
+import nathan from '../assets/execs/nathan.svg'
+import karren from '../assets/execs/karren.svg'
+import josh from '../assets/execs/josh.svg'
+
+const execs = [
+  {
+    src: praneer,
+    width: '100px',
+    height: '100px',
+    name: 'Praneer.png',
+    link: 'https://www.linkedin.com/in/praneershrest/'
+  },
+  {
+    src: grace,
+    width: '100px',
+    height: '100px',
+    name: 'Grace.gif',
+    link: 'https://www.linkedin.com/in/grace-lu0/'
+  },
+  {
+    src: noble,
+    width: '100px',
+    height: '100px',
+    name: 'Noble.gif',
+    link: 'https://www.linkedin.com/in/nobletan/'
+  },
+  {
+    src: vivian,
+    width: '100px',
+    height: '100px',
+    name: 'Vivian.tiff',
+    link: 'https://www.linkedin.com/in/vivian-wong-114032164'
+  },
+  {
+    src: naomi,
+    width: '100px',
+    height: '100px',
+    name: 'Naomi.gif',
```

Comment:
don't use .gif or .svg, that looks off, any proper image formats like .jpeg, .png, .tiff, .cr2 should be fine